charts/k3s/templates/syncer.yaml

```diff
@@ -370,6 +373,9 @@ spec:
               - --datastore-certfile=/data/pki/apiserver-etcd-client.crt
               - --datastore-keyfile=/data/pki/apiserver-etcd-client.key
               {{- end }}
```

Comment:
This addition may not be correct as these are the args for the vcluster api-server.